tests/e2e/test_api.py

```diff
@@ -64,16 +62,15 @@ def test_valid_question() -> None:
         json={"conversation_id": conversation_id, "query": "what is kubernetes?"},
         timeout=90,
     )
-    print(vars(response))
     assert response.status_code == requests.codes.ok
     json_response = response.json()
     json_response["conversation_id"] == conversation_id
     # assuming the response will be consistent
     assert (
-        "Kubernetes is an open source container orchestration tool"
+        "The following response was generated without access to reference content"
         in json_response["response"]
     )
     assert (
-        "The following response was generated without access to reference content:"
-        not in json_response["response"]
```

Comment:
Ok so it seems that the index is available in e2e, while I don't have it locally. What would be the right approach here?

The second update tool->platform seems to be valid though.

Reply:
Yes, we do have an [index](https://github.com/openshift/lightspeed-service/blob/main/scripts/test-e2e-standalone.sh#L23) deployed as a part of e2e test in PROW. But if we want to distinguish our test asserts based on local and PROW, we can use these [envs](https://docs.prow.k8s.io/docs/jobs/#job-environment-variables) probably the `JOB_TYPE`.

Reply:
I guess it might not be a good thing to have different asserts based on env, but on the other side, e2e are bit special here. So I can't say I'm against it. :)

We need some consensus if this is the way we want to proceed with our e2e tests. Otherwise, we need to update the dev process to ensure the dev env is the same as in CI for the e2e test.

So @bparees @tisnik @xrajesh WDYT?

Reply:
The dev process should be to run `scripts/test-standalone-e2e.sh` which pulls down and sets up the RAG content.

In general going forward we should be assuming the RAG content is present during testing and devs will need to get used to having it (it's not that hard to setup once and reference from your config yaml)